eng/common/docgeneration/Generate-DocIndex.ps1

```diff
@@ -177,8 +176,6 @@ function UpdateDocIndexFiles {
     # Update main.js var lang
     $mainJsContent = Get-Content -Path $MainJsPath -Raw
     $mainJsContent = $mainJsContent -replace "var SELECTED_LANGUAGE = ''", "var SELECTED_LANGUAGE = '$lang'"
-    # Update main.js var index html
```

Comment:
Is this not needed or used by any other language?

Reply:
Since it provides default value, other lang do not initiate it when use.